resources/lang/epo.ini

```diff
@@ -0,0 +1,114 @@
+session.undo.left = "Restas{%0} da malfaroagoj"
+session.redo.none = "Neniu por refari"
+session.redo.left = "Restas {%0} da refaroagoj"
+session.brush.added = "Aldonis {%0} al sesio"
+session.brush.deleted = "Foriĝis {%0} (UUID {%1})"
+session.brush.removed = "Formovis {%0} (UUID {%1})"
+session.language.set = "Agordis lingvon al {%0}"
+session.language.notfound = "Lingvo {%0} ne trovita, uzante defaŭlton"
+; task
+task.copy.success = "Sukcesis neinsinkrona kopio, daŭris {%0}, kopiis {%1} da blokoj el{%2}."
+task.count.success = "Sukcesis neinsinkrona analizo, daŭris {%0}"
+task.count.result = "{%0} da blokoj trovitaj el totalo de {%1} blocks"
+task.replace.success = "Sukcesis neinsinkrona enspacigado, daŭris {%0}, ŝanĝis {%1} da blokoj el{%2}."
+task.replace.success = "Sukcesis neinsinkrona interŝanĝo, daŭris {%0}, ŝanĝis {%1} da blokoj el{%2}."
+task.revert.undo.success = "Sukcesis neinsinkrona malfarado, daŭris {%0}, ŝanĝis {%1} da blokoj el{%2}."
+task.revert.redo.success = "Sukcesis neinsinkrona refarado, daŭris {%0}, ŝanĝis {%1} da blokoj el{%2}."
+; flags
+flags.keepexistingblocks = "Konservu jamaj blokoj"
+flags.keepair = "Konservu aero"
+flags.hollow = "Kava"
+flags.hollowclosed = "Kava kun ŝtopitaj ekstremaĵoj"
+flags.natural = "Natura"
+; tools
+; wand tool
+tool.wand = "Sorĉbastono"
+tool.wand.lore.1 = "Alklaku maldekstren por agordi la pozicio 1 de elektaro"
+tool.wand.lore.2 = "Alklaku dekstren por agordi la pozicio 2 de elektaro"
+tool.wand.lore.3 = "Use //togglewand to toggle it's functionality"
+tool.wand.lore.3 = "Baskulu ĝin per //togglewand"
+tool.wand.disabled = "La sorĉbastono estas malaktivigita. Baskulu ĝin per //togglewand"
+tool.wand.setenabled = "La sorĉbastono nun estas {%0}!"
+; debug tool
+tool.debug = "Erarserĉilo"
+tool.debug.lore.1 = "Alklaku maldesktre por ricevi informon"
+tool.debug.lore.2 = "kiel la nomo kaj damaĝovaloro de bloko"
+tool.debug.lore.3 = "Uzu //toggledebug por baskuli la funkciado"
+tool.debug.disabled = "La erarserĉilo estas malativigita. Reaktivigu ĝin per //toggledebug"
+tool.debug.setenabled = "La erarserĉilo nun estas {%0}!"
+; flood tool
+ui.flood.title = "Plenigomenuo"
+ui.flood.options.limit = "Blokolimo"
+ui.flood.options.blocks = "Blokoj"
+ui.flood.options.blocks.placeholder = "Blokoj apartigita per punktokomo"
+ui.flood.options.label.infoapply = "Alklaku la "Submit" butono por apliku"
+; brush tool
+ui.brush.title = "Brosomenuo"
+ui.brush.content = "Broso ĉefmenuo"
+ui.brush.create = "Kreu nova"
+ui.brush.getsession = "Akiru sesiobroso"
+ui.brush.edithand = "Redaktu broso enmane"
+; brush settings
+ui.brush.settings.title = "{%0} brosagordoj"
+; brush options
+ui.brush.options.blocks = "Blokoj"
+ui.brush.options.blocks.placeholder = "Exemple: 1:1,2,tnt,log:12"
+ui.brush.options.diameter = "Diametro"
+ui.brush.options.width = "Larĝeco"
+ui.brush.options.height = "Alteco"
+ui.brush.options.depth = "Profundeco"
+ui.brush.options.flags = "Add flags?"
```

Comment:
This might need translations - it actually means "Add options" or "Add extra options" or "Add modifiers" :P